mytimer/functions.py

```diff
@@ -459,16 +459,29 @@ def run_timer(args):
         show_faces_list()
     elif args.programs_list:
         show_programs_list()
-    elif args.program == "pomodoro":
-        short_break_params = load_params(args, program="pomodoro-short-break", is_break=True)
-        long_break_params = load_params(args, program="pomodoro-long-break", is_break=True)
-        pomodoro_timer(
-            timer_func,
-            params=params,
-            long_break_params=long_break_params,
-            short_break_params=short_break_params)
-    elif args.program in ["52-17", "112-26", "animedoro"]:
-        break_params = load_params(args, is_break=True)
-        two_step_timer(timer_func, params1=params, params2=break_params)
     else:
-        timer_func(**params)
+        timer_round = 0
+        while timer_round < args.repeat or args.repeat == -1:
+            if args.program == "pomodoro":
+                short_break_params = load_params(args, program="pomodoro-short-break", is_break=True)
+                long_break_params = load_params(args, program="pomodoro-long-break", is_break=True)
+                pomodoro_timer(
+                    timer_func,
+                    params=params,
+                    long_break_params=long_break_params,
+                    short_break_params=short_break_params)
+            elif args.program in ["52-17", "112-26", "animedoro"]:
+                break_params = load_params(args, is_break=True)
+                two_step_timer(timer_func, params1=params, params2=break_params)
+            else:
+                timer_func(**params)
+
+            end_round_message = END_ROUND_MESSAGE.format("{}/{}".format(timer_round + 1, args.repeat))
+            if args.repeat == -1:
+                end_round_message = END_ROUND_MESSAGE.format(timer_round + 1)
+            if timer_round < args.repeat - 1 or args.repeat == -1:
```

Comment:
Why `or args.repeat == -1`?
Under infinite loop mode,`end_round_message` will never be printed.

Reply:
OK. I will fix it then.